### PR TITLE
update error-chain to 0.7

### PR DESCRIPTION
**Pull request by pfernie**
Changes the error_chain! invocation to the new style used in error-chain 0.7, and bumps the dependency version

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ quick-xml = "0.4.2"
 log = "0.3.6"
 encoding = "0.2.33"
 byteorder = "0.5.3"
-error-chain = "0.6.2"
+error-chain = "0.7"
```
Comment (tafia) [inline Cargo.toml]: If possible I'd prefer `0.7.2`.

File: src/errors.rs [hunk 2]
```diff
@@ -8,23 +8,23 @@
 use quick_xml::error::Error as XmlError;
 
 error_chain! {
-    types {
-        Error, ErrorKind, Result;
-    }
+//    types {
+//        Error, ErrorKind, ResultExt, Result;
+//    }
```
Comment (tafia) [inline src/errors.rs]: Could you remove commented lines (it is confusing as `ResultExt` won't mean anything).

**Comment by pfernie** [comment]
I updated the PR per your requests; I removed all commented code in errors.rs as it's just the example invocation from error-chain docs, and they are out of date w.r.t. 0.7.2.

**Comment by tafia** [comment]
Thanks! 
Publishing 0.3.3

**Comment by tafia** [comment]
> Publishing 0.3.3

upgraded byteorder to 1.0.0 too

**Pull request merged**